api/src/main/java/io/strimzi/api/kafka/model/CruiseControlSpec.java

```diff
@@ -22,20 +24,37 @@
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({
-        "replicas", "image", "tlsSidecar"})
+        "replicas", "image", "storage",
+        "listeners", "config",
+        "livenessProbe", "readinessProbe",
+        "jvmOptions", "resources",
+        "metrics", "logging", "tlsSidecar", "template"})
 @EqualsAndHashCode
 public class CruiseControlSpec implements UnknownPropertyPreserving, Serializable {
     private static final long serialVersionUID = 1L;
 
+    public static final String FORBIDDEN_PREFIXES = "bootstrap.servers, zookeeper., ssl., security., failed.brokers.zk.path,"
+        + "webserver.http.port, webserver.http.address, webserver.api.urlprefix, metric.reporter.sampler.bootstrap.servers, metric.reporter.topic, metric.reporter.topic.pattern,"
+        + "partition.metric.sample.store.topic, broker.metric.sample.store.topic, capacity.config.file, skip.sample.store.topic.rack.awareness.check, cruise.control.metrics.topic"
+        + "cruise.control.metrics.reporter.bootstrap.servers, sasl. interceptor.classes";
+    public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "";
+
     private int replicas;
     private String version;
     private String image;
     private TlsSidecar tlsSidecar;
-
     private ResourceRequirements resources;
-
+    private Probe livenessProbe;
+    private Probe readinessProbe;
+    private JvmOptions jvmOptions;
+    private Logging logging;
+    private Map<String, Object> metrics;
```

Comment:
So it seems that CC provides metrics through JMX. Because we need to Prometheus JMX exporter configuration here, do you have an example for it? I am thinking about leaving these metrics out of CC for now.

Reply:
I don't have an example yet, I still need to wire this up, but I can do that in a later PR. Should I strip this line `private Map<String, Object> metrics;` out of the CruiseControlSpec in the meantime? Or should I leave it in?

Reply:
I would remove it for now